lib/models/user_details.dart

```diff
@@ -118,3 +118,18 @@ class FamilyData {
     );
   }
 }
+
+class FilesCount {
+  final Map<int, int> filesCount;
+  FilesCount(this.filesCount);
+
+  int get total => images + videos + livePhotos + (filesCount[-1] ?? 0);
+
+  int get photos => images + livePhotos;
+
+  int get images => filesCount[0] ?? 0;
+
+  int get videos => filesCount[1] ?? 0;
+
+  int get livePhotos => filesCount[2] ?? 0;
```

Comment:
instead of hard coding the numbers here, should we say getInt(FileType.video) etc etc?